python/doc/examples/graphics/graphics_basics.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Creating graphics\n",
+    "# A quick start guide to graphics \n",
```

Comment:
I am not sure about the word "graphics", which is heavily used throughout the notebook. I feel that in most places, it could be replaced by "graph", which is after all the name of the illustrated class.

Reply:
I suggest to create a separate review on this topic, so that the appropriate wording can be chosen at the appropriate place. Please wait until the PR is integrated, then suggest the changes.